src/rules/disable-social-signup.js

```diff
@@ -8,7 +8,8 @@
  *
  */
 
-function (user, context, callback) {
+function disableSocialSignups(user, context, callback) {
+  const auth0 = require("auth0");
```

Comment:
This might be a problem... how does this currently work in production? AFAIK, this is currently [instantiated upstream ](https://github.com/auth0/auth0-server/blob/master/lib/auth/executeRules.js#L150) and preconfigured similar to the below 
```
const auth0 = require('auth0').ManagementClient;
var management = new ManagementClient({
  token: '{YOUR_API_V2_TOKEN}',
  domain: '{YOUR_ACCOUNT}.auth0.com'
});
```

Reply:
Oh awesome! Nice catch. Now that I look at the `rulez-api` again, it's hanging a capital "A" `Auth0` on the global object, not auth0. I'll revisit this.